docs/maya/simple_setup.md

```diff
@@ -317,3 +317,46 @@ Refer to this [section](simshape#muscle-activations) to see how to use Muscle Pa
 </figure>
 
 In the attribute editor of the AdnSimshape deformer, under the *Muscles Activation* section, the *Plug Values* will be enabled as a new valid *Activation Mode* option. To better visualize activations, press the ![AdnMuscle](images/adn_simshape_debugger.png){style="width:4%"} shelf button or go to AdonisFX Menu > Simshape (on the *Edit* group) > *Activations Debugger*.
+
+## AdnSkinMerge
+
+To create a basic scenario using the AdnSkinMerge deformer, start with a scene with the following elements:
+
+  - An animation mesh with deformation.
+  - A skin mesh with an AdnSkin deformer applied and configured.
```

Comment:
If the user has already an alembic with the results of the skin simulation, should he/she also be able to use AdnSkinMerge? If so we could add something like:

A skin mesh with an AdnSkin deformer applied and configured or the cached geometry resulting from the skin simulation.

Reply:
In this case I'd leave it as it is because we are referrencing this specific scenario we are setting up. I will update the skin_merge.md page with this though.